main.py

```diff
@@ -97,6 +101,10 @@ def __show_help_screen(self, signal_emitter=None):
     def __show_prev_screen(self, signal_emitter=None):
         self.__set_overlay(self.prev)
 
+    def show_game_over_screen(self, signal_emitter=None):
```

Comment:
this should be private (__show_game_over_screen)

Reply:
im calling it from the state manager it has to be public